src/Admin/ReenableCssTransientCachingAjaxAction.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Class ReenableCssTransientCachingAjaxAction.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AMP_Options_Manager;
+use AmpProject\AmpWP\Option;
+
+/**
+ * Base class to define a new AJAX action.
+ *
+ * @package AmpProject\AmpWP
+ */
+final class ReenableCssTransientCachingAjaxAction {
+
+	/**
+	 * Action to use for enqueueing the JS logic at the backend.
+	 *
+	 * @var string
+	 */
+	const BACKEND_ENQUEUE_ACTION = 'admin_enqueue_scripts';
+
+	/**
+	 * AJAX action name to use.
+	 *
+	 * @var string
+	 */
+	const AJAX_ACTION = 'amp_reenable_css_transient_caching';
+
+	/**
+	 * Selector to attach the click handler to.
+	 *
+	 * @var string
+	 */
+	const SELECTOR = 'a.reenable-css-transient-caching';
+
+	/**
+	 * Register the AJAX action with the WordPress system.
+	 */
+	public function register() {
+		add_action( static::BACKEND_ENQUEUE_ACTION, [ $this, 'register_ajax_script' ] );
+		add_action( 'wp_ajax_' . self::AJAX_ACTION, [ $this, 'reenable_css_transient_caching' ] );
+	}
+
+	/**
+	 * Register the AJAX logic.
+	 *
+	 * @param string $hook_suffix Hook suffix to identify from what admin page the call is coming from.
+	 */
+	public function register_ajax_script( $hook_suffix ) {
+		if ( 'site-health.php' !== $hook_suffix ) {
+			return;
+		}
+
+		$script = <<< JS_SCRIPT
```

Comment:
If we don't use string interpolation, we can use a nowdoc instead of a heredoc:
```suggestion
		$script = <<< 'JS_SCRIPT'
```